firmware/application/apps/ui_freqman.cpp

```diff
@@ -96,7 +97,7 @@ void FreqManBaseView::focus() {
     if (error_ == ERROR_ACCESS) {
         nav_.display_modal("Error", "File access error", ABORT);
     } else if (error_ == ERROR_NOFILES) {
-        nav_.display_modal("Error", "No database files\nin /FREQMAN", ABORT);
+        nav_.display_modal("Error", "No database files\nin either /FREQMAN\nor /SYS/FREQMAN", ABORT);
```

Comment:
Instead of hard-coding the paths in this string, I'd recommend using the path strings in file_path.cpp